docs/concepts.md

```diff
@@ -51,8 +50,324 @@ The newly provisioned resource is automatically bound to the resource claim.
+An application developer simply has to specify their general need for a database such as MySQL, 
+without any specific knowledge of what environment that database will run in or even what 
+specific type of database it will be at runtime.
+
+The cluster administrator specifies a resource class that acts as a template with the 
+implementation details and policy specific to the environment that the generic MySQL resource is being deployed to.
+This enables the database to be dynamically provisioned at deployment time without the 
+application developer needing to know any of the details, which promotes portability and reusability.
+
+Future support for additional managed services.
+
+### Storage Services
+Support for S3, Buckets, and Azure Blob storage.
+
+Future support for additional managed services.
+
+### Networking Services
+Support for networks, subnets, and firewall rules.
+
+Future support for additional managed services.
+
+### Load Balancing Services
+Future support.
+
+### Cloud DNS Services
+Future support.
+
+### Advanced Networking Connectivity Services
+Future support.
+
+### Big Data Services
+Future support.
+
+### Machine Learning Services
+Future support.
+
+## Crossplane Stacks
+Stacks extend Crossplane with new functionality. 
+
+See [Stacks](#stacks).
+
+## Crossplane Workloads 
+Crossplane includes an extensible workload scheduler that observes application 
+policies to select a suitable target cluster from a pool of available clusters. 
+The workload scheduler can be customized to consider a number of criteria including 
+capabilities, availability, reliability, cost, regions, and performance while 
+deploying workloads and their resources. Complex workloads can be modeled as a `KubernetesApplication`.
+
+## Crossplane Clusters 
+Crossplane supports dynamic provisioning of managed
+Kubernetes clusters from a single control plane with consistent multi-cluster
+best-practice configuration and secure connectivity between target Kubernetes
+clusters and the managed services provisioned for applications. Managed
+Kubernetes clusters can be dynamically provisioned with a `KubernetesCluster`.
+
+# Glossary
+
+## Kubernetes
+Crossplane is built on the Kubernetes API machinery as a platform for declarative management.
+We rely on common terminology from the [Kubernetes Glossary][kubernetes-glossary] where possible,
+and we don't seek to reproduce that glossary here.
+
+[kubernetes-glossary]: https://kubernetes.io/docs/reference/glossary/?all=true
+However we'll summarize some commonly used concepts for convenience.
+
+### CRD
+A standard Kubernetes Custom Resource Definition (CRD), which defines a new type of resource that can be managed declaratively. 
+This serves as the unit of management in Crossplane.
+The CRD is composed of spec and status sections and supports API level versioning (e.g., v1alpha1)
+
+### Controller
+A standard Kubernetes Custom Controller, providing active control loops that own or more CRDs. 
```

Comment:
Should be "that own one or more CRDs" and I think I would actually prefer "that watch one ore more CRDs" instead